pkg/plugin/installer/base_test.go

```diff
@@ -32,6 +32,10 @@ func TestPath(t *testing.T) {
 			source:         "https://github.com/jkroepke/helm-secrets",
 			helmPluginsDir: "/helm/data/plugins",
 			expectPath:     "/helm/data/plugins/helm-secrets",
+		}, {
+			source:         "https://github.com/jkroepke/helm-secrets",
+			helmPluginsDir: "/helm/data/plugins:/remote/helm/data/plugins",
```

Comment:
This test is going to fail on Windows which is one of our supported environments. We need to use the correct path list separator for each environment. See https://pkg.go.dev/path/filepath#pkg-constants

Reply:
I'm not sure where I have to add docs. How should I add it in the current MR? Could you please give me a link where I can find how to do this?